### DISCUSSION TITLE
よろしくお願いします！！

### DISCUSSION THREAD

**Discussion by unnnyou** [UNNNYOU/Project-RQB#1] [アナウンス]
## ようこそ！
  私たちは、コミュニティの他のメンバーとつながる場としてディスカッションを活用しています。次のことにご協力ください。
 * 疑問に思っていることを質問してください。
 * アイデアを共有しましょう。
 * 他のコミュニティメンバーと交流しましょう。
 * 他の人を歓迎し、心を開いてください。これは私たちが一緒に築くコミュニティであることを忘れないでください💪。

  まず、以下にコメントして自己紹介をし、このコミュニティで何をしているかを教えてください

**Comment by topi-log** [comment]
### はじめまして！
とぴです。コメントテストです

**Comment by neko967** [comment]
見れましたー！
